cockpit_api/src/main/java/com/cockpit/api/service/jiragateway/JiraGatewayService.java

```diff
@@ -1,18 +1,23 @@
 package com.cockpit.api.service.jiragateway;
 
+import com.cockpit.api.exception.ResourceNotFoundException;
```

Comment:
Unnecessary imports should be removed: import com.cockpit.api.exception.ResourceNotFoundException;